src/index.html

```diff
@@ -6,9 +6,50 @@
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="stylesheet" href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap">
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header__big">
+      <div class="logo__div">
+        <a href="#logo" class="moyologo ">
+          <img src="images/logo.png" alt="sorry">
+        </a>
+      </div>
+      <nav class="navigation__big">
+        <ul class="ul__big">
```

Comment:
Don't use general tag names in class names (like ul, li, a, p etc) However, header, nav or footer are correct to use, because they have a meaning.
```suggestion
        <ul class="nav__list">
```